extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -855,6 +913,64 @@ export class SystemLayer {
     });
   }
 
+  /**
+   * Reacts to a custom configuration change coming from the player level.
+   * Updates UI to match configuration described by publisher.
+   * @param {!Array<!Object>} controls
+   * @private
+   */
+  onCustomControls_(controls) {
+    if (controls.length <= 0) {
+      return;
+    }
+
+    controls.forEach((control) => {
+      if (!control.name) {
+        return;
+      }
+
+      const defaultConfig = CONTROL_DEFAULTS[control.name];
+
+      if (!defaultConfig) {
+        return;
+      }
+
+      const element = scopedQuerySelector(
+        this.getShadowRoot(),
+        defaultConfig.selector
+      );
+
+      if (control.visibility === 'hidden') {
+        this.vsync_.mutate(() => {
+          element.classList.add('i-amphtml-story-ui-hide-button');
+        });
+      }
+
+      if (!control.visibility || control.visibility === 'visible') {
+        this.vsync_.mutate(() => {
+          element.classList.remove('i-amphtml-story-ui-hide-button');
+        });
+      }
+
+      if (control.position === 'start') {
+        const startButtonContainer = this.systemLayerEl_.querySelector(
+          '.i-amphtml-story-system-layer-buttons-start-position'
+        );
+
+        this.vsync_.mutate(() => {
+          element.parentElement.removeChild(element);
+          startButtonContainer.appendChild(element);
+        });
+      }
+
+      if (control.icon) {
+        setImportantStyles(dev().assertElement(element), {
+          'background-image': `url(${control.icon})`,
```

Comment:
Let's clarify a bit more what the `icon` is. Do we want it to always be a URL, or could it be a SVG inlined? Would be cool to have both I assume, how do we allow that?